src/index.html

```diff
@@ -22,6 +22,302 @@
+          <p class="card__text card__text--price">Price:</p>
+          <p class="card__text card__text--costs">$2,199</p>
+        </div>
+
+        <a class="card__link">BUY</a>
+      </div>
+
+      <div class="card">
+        <div class="card__image"></div>
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__text card__text--product-code">Product code: 195434</p>
+
+        <div class="card__row card__row--feedback">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__text card__text--reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__row card__row--price">
+          <p class="card__text card__text--price">Price:</p>
+          <p class="card__text card__text--costs">$2,199</p>
+        </div>
+
+        <a class="card__link">BUY</a>
+      </div>
+
+      <div class="card">
+        <div class="card__image"></div>
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__text card__text--product-code">Product code: 195434</p>
+
+        <div class="card__row card__row--feedback">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__text card__text--reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__row card__row--price">
+          <p class="card__text card__text--price">Price:</p>
+          <p class="card__text card__text--costs">$2,199</p>
+        </div>
+
+        <a class="card__link">BUY</a>
+      </div>
+
+      <div class="card">
+        <div class="card__image"></div>
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__text card__text--product-code">Product code: 195434</p>
+
+        <div class="card__row card__row--feedback">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__text card__text--reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__row card__row--price">
+          <p class="card__text card__text--price">Price:</p>
+          <p class="card__text card__text--costs">$2,199</p>
+        </div>
+
+        <a class="card__link">BUY</a>
+      </div>
```

Comment:
All product cards currently display the same details. If this is not intentional, consider using a templating approach or dynamically generating the content to ensure each card has unique information.